pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -107,9 +106,10 @@ in rec {
       };
     } ./python-imports-check-hook.sh) {};
 
-  pythonNamespacesHook = callPackage ({}:
+  pythonNamespacesHook = callPackage ({ findutils }:
     makeSetupHook {
       name = "python-namespaces-hook.sh";
+      deps = [ findutils ];
```

Comment:
not necessary, please undo this change. findutils is already substituted below. without this change this pr doesn't rebuild anything so can go to master

Reply:
See also https://github.com/NixOS/nixpkgs/pull/192624